lib/psci/psci_common.c

```diff
@@ -893,7 +893,7 @@ void psci_print_power_domain_map(void)
 
 	INFO("PSCI Power Domain Map:\n");
 	for (idx = 0; idx < (PSCI_NUM_PWR_DOMAINS - PLATFORM_CORE_COUNT);
-							idx++) {
+					idx = SPECULATION_SAFE_VALUE(idx + 1)) {
```

Comment:
Again, I'm struggling to see the threat as this is also in the cold boot path and is limited to speculatively reading off the end of the psci_non_cpu_pd_nodes[] array.

Reply:
As discussed offline, we have revised decisions around cold boot path.